Libs/RaptureAddins/ElasticSearch/src/main/java/rapture/elasticsearch/ElasticSearchSearchRepository.java

```diff
@@ -217,9 +217,22 @@ public void dropIndexForRepo(String repoName) {
 
     }
 
+    private static String[] _all = null;
+
+    public static String[] allTypes() {
```

Comment:
this code should be in the auto-generated SearchRepoType class in RaptureAPI.  Furthermore you can do it 1 line statically e.g.

    public static String[] _all = SearchRepoType.values.toArray(new String[SearchRepoType.values.size()]);